File: src/main/java/com/back/mozu/domain/reservation/dto/WaitingRoomResponseDto.java
package com.back.mozu.domain.reservation.dto;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class WaitingRoomResponseDto {
    private UUID reservationId;
    private String status;
    private Integer queueNumber;
    private Integer estimatedWaitMinutes;
}

File: src/main/java/com/back/mozu/domain/reservation/dto/WaitingRoomResponseDto.kt
package com.back.mozu.domain.reservation.dto

import java.util.UUID

data class WaitingRoomResponseDto(
    val reservationId: UUID? = null,
    val status: String? = null,
    val queueNumber: Int? = null,
    val estimatedWaitMinutes: Int? = null,
)
